lib/kitchen/driver/openstack_version.rb

```diff
@@ -22,6 +22,6 @@ module Kitchen
   #
   # @author Jonathan Hartman <j@p4nt5.com>
   module Driver
-    OPENSTACK_VERSION = '2.1.0'
+    OPENSTACK_VERSION = '2.1.1'
```

Comment:
No worries on this.   This was for local testing reasons.  Wanted to know I was installing my change.  Will revert in final PR.